test/unit/viewModels/YouCantUseServiceViewModelSpec.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.viewModels
+
+import base.UnitSpec
+import play.api.test.FakeRequest
+import uk.gov.hmrc.auth.core.AffinityGroup
+import uk.gov.hmrc.eoricommoncomponent.frontend.viewModels.YouCantUseServiceViewModel
+import uk.gov.hmrc.eoricommoncomponent.frontend.views.ServiceName.longName
+import util.ControllerSpec
+
+class YouCantUseServiceViewModelSpec extends UnitSpec with ControllerSpec {
+  implicit val fakeRequest = FakeRequest()
+
+  "YouCantUseServiceViewModel firstParagraph" should {
+    "return the correct first paragraph for the 'Agent' user type" in {
+      val name   = Option(AffinityGroup.Agent)
+      val result = YouCantUseServiceViewModel.firstParagraph(name)(messages, fakeRequest)
+      result shouldBe messages("cds.you-cant-use-service-agent.para1")
+    }
+    "return the correct first paragraph for other user types" in {
+
+      val result = YouCantUseServiceViewModel.firstParagraph(None)(messages, fakeRequest)
+      result shouldBe messages("cds.you-cant-use-service-standard-organisation.para1", longName)
+    }
```

Comment:
We aren't testing all other user types here, only None - can just use a Seq[AffinityGroup] here and pass in the values we want to test to make sure that if anything changes in the future that this functionality is still working

Reply:
As a side note, the types that need adding are Individual and Organisation